src/main/java/com/dragonballs/services/authentication/JwtTokenUtil.java

```diff
@@ -0,0 +1,115 @@
+package com.dragonballs.services.authentication;
+
+import com.dragonballs.entities.User;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Clock;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.impl.DefaultClock;
+import org.springframework.stereotype.Component;
+
+import java.io.Serializable;
+import java.util.Base64;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Component
+public class JwtTokenUtil implements Serializable {
+
+    //private String secret = "testSecret";
+    private final String secret = Base64.getEncoder().encodeToString("TestSecret".getBytes());
```

Comment:
Try to move your secret to .properties file, at least - ideally - your secret, for security reasons, should never appear in git - it should be stored in a file which wouldn't be tracked by GIT - you'd simply have it on your computer.